install-little-backup-box.sh

```diff
@@ -109,6 +111,18 @@ case $CHOICE in
         echo "@reboot sudo /home/"$USER"/little-backup-box/scripts/backup.sh camera internal > /home/"$USER"/little-backup-box.log 2>&1"
     } | crontab
     ;;
+5)
+    crontab -l | {
+        cat
+        echo "@reboot sudo /home/"$USER"/little-backup-box/scripts/backup.sh ios internal > /home/"$USER"/little-backup-box.log 2>&1"
+    } | crontab
+    ;;
+6)
+    crontab -l | {
+        cat
+        echo "@reboot sudo /home/"$USER"/little-backup-box/scripts/backup.sh ios internal > /home/"$USER"/little-backup-box.log 2>&1"
```

Comment:
I think `backup.sh ios internal` in this line should be `backup.sh ios external`?